### PR TITLE
[9.4.x] [PREVIEW] ISPN-11373 XSite backup commands should be sent from a blocking thread

### PR DESCRIPTION
**Pull request by danberindei**
https://issues.redhat.com/browse/ISPN-11373

### CODE REVIEW

File: core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java [hunk 1]
```diff
@@ -326,13 +326,22 @@ public BackupResponse backupRemotely(Collection<XSiteBackup> backups, XSiteRepli
          DeliverOrder order = xsb.isSync() ? DeliverOrder.NONE : DeliverOrder.PER_SENDER;
          long timeout = xsb.getTimeout();
          try {
-            sendCommand(recipient, command, request.getRequestId(), order, rsvp, false, false);
+            Message message = new Message(toJGroupsAddress(recipient));
+            marshallRequest(message, command, request.getRequestId());
+            setMessageFlags(message, order, rsvp, false);
+
+            remoteExecutor.submit(() -> {
```
Comment (pruivo) [inline core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java]: -1
sending order is important. You need to use a single thread to send the request or you risk it to be reordered.

Comment (danberindei) [reply core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java]: Oops :)

Comment (danberindei) [reply core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java]: I added `ActionSequencer` when the commands need to be ordered

**Comment by pruivo** [comment]
integrated! thanks @danberindei !

**Pull request merged**